src/index.html

```diff
@@ -1,12 +1,422 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="logo-link">
+        <img src="./images/logo.png" alt="Logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__li">
+            <a class="nav__link is-active" href="#apple">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__li">
```

Comment:
`nav__li` isn't the best classname, probably `nav__item` will be better
don't use tag names in class names